contrib/get-drat2er

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+source "$(dirname "$0")/get-script-header.sh"
+if [ -e drat2er ]; then
+  echo 'error: file or directory "drat2er" exists; please move it out of the way.' >&2
+  exit 1
+fi
+
+git clone https://github.com/alex-ozdemir/drat2er.git
+
+cd drat2er
+
+git checkout api
+
+mkdir build
+
+cd build
+
+cmake ..
+
+make install
```

Comment:
A comment on best practices. You should not set `CMAKE_INSTALL_PREFIX` in your CMakeLists.txt by default to `${CMAKE_BINARY_DIR}/install`, since the user will never be able to install it to a different directory (or install it on the system). The cmake-way is to let the user specify a custom ` CMAKE_INSTALL_PREFIX` when configuring: `cmake .. -DCMAKE_INSTALL_PREFIX=$(pwd)/build/install`.